enable/viewable.py

```diff
@@ -16,6 +16,7 @@ class Viewable(HasTraits):
     def request_redraw(self):
         # This overrides the default Component request_redraw by asking
         # all of the views to redraw themselves.
+        return
```

Comment:
You can also just change the comment to a docstring so that the `return` can still be removed.